src/components/sideContent/sideContent.module.scss

```diff
@@ -0,0 +1,25 @@
+.headerContent {
```

Comment:
We should add a bit of margin between the subregion elements. Tapahtumat.hel.fi has `margin-bottom: var(--spacing-l);` on each wrapper div.